PROJECT

```diff
@@ -19,7 +18,6 @@ resources:
     webhookVersion: v1
 - api:
     crdVersion: v1
-    namespaced: false
```

Comment:
yeah it looks like it's false if its not there because it was removed by kubebuilder. Nevertheless I added it [here](https://github.com/kubewarden/kubewarden-controller/pull/247/commits/84e2dd165f0d02ea77d1116f6ab40212740a3d57) since I also think it should be there